.circleci/config.yml

```diff
@@ -201,50 +215,50 @@ jobs:
     parallelism: 1
     executor: linux-amd64
     steps:
-    - checkout
-
-    - attach_workspace:
-        at: .
-
-    - install_go:
-        executor: linux-amd64
-
-    - run:
-        name: Build tarball
-        command: |
-          echo "$PRIVATE_PEM_B64" | base64 --decode > /tmp/private.pem
-          echo "$PUBLIC_PEM_B64" | base64 --decode > /tmp/public.pem
-          export PRIVATE_KEY_PASSPHRASE="$(echo $PRIVATE_KEY_PASSPHRASE_B64 | base64 --decode)"
-          # Prevent rebuilding web ui, we should have already attached it
-          find webui -exec touch -c '{}' +
-          GOOS=linux GOARCH=amd64 make build-bacalhau-tgz
-          GOOS=linux GOARCH=arm64 make build-bacalhau-tgz
-          GOOS=darwin GOARCH=amd64 make build-bacalhau-tgz
-          GOOS=darwin GOARCH=arm64 make build-bacalhau-tgz
-          GOOS=linux GOARCH=armv6 make build-bacalhau-tgz
-          GOOS=linux GOARCH=armv7 make build-bacalhau-tgz
-          GOOS=windows GOARCH=amd64 make build-bacalhau-tgz
-
-    - persist_to_workspace:
-        root: dist/
-        paths:
-        - "*.tar.gz"
-        - "*.sha256"
-
-    - store_artifacts:
-        path: dist/
+      - checkout
+
+      - attach_workspace:
+          at: .
+
+      - install_go:
+          executor: linux-amd64
+
+      - run:
+          name: Build tarball
+          command: |
+            echo "$PRIVATE_PEM_B64" | base64 --decode > /tmp/private.pem
+            echo "$PUBLIC_PEM_B64" | base64 --decode > /tmp/public.pem
+            export PRIVATE_KEY_PASSPHRASE="$(echo $PRIVATE_KEY_PASSPHRASE_B64 | base64 --decode)"
+            # Prevent rebuilding web ui, we should have already attached it
+            find webui -exec touch -c '{}' +
+            GOOS=linux GOARCH=amd64 make build-bacalhau-tgz
+            GOOS=linux GOARCH=arm64 make build-bacalhau-tgz
+            GOOS=darwin GOARCH=amd64 make build-bacalhau-tgz
+            GOOS=darwin GOARCH=arm64 make build-bacalhau-tgz
+            GOOS=linux GOARCH=armv6 make build-bacalhau-tgz
+            GOOS=linux GOARCH=armv7 make build-bacalhau-tgz
+            GOOS=windows GOARCH=amd64 make build-bacalhau-tgz
+
+      - persist_to_workspace:
+          root: dist/
+          paths:
+            - "*.tar.gz"
+            - "*.sha256"
+
+      - store_artifacts:
+          path: dist/
 
   build_webui:
-    executor: node/default
+    executor: linux-amd64
     steps:
       - checkout
-      - node/install-packages:
-          app-dir: webui
```

Comment:
This used to install the version of node specified in the package.json automatically. So we need to install a specific version of node now corresponding to the one in `.tool-versions`.

Reply:
done